### PR TITLE
Adds a footer using Lit Web Components

### PR DESCRIPTION
**Pull request by admeeer**
## Summary of Changes
Adds a footer, using Lit web components.
These web components are pretty cool and while the footer is pretty bare-bones, it can be customized to anything. I mainly wanted to use Lit because I really, really didn't like the heavy amount of boilerplate that HTML uses. 

## Note
Because of the nature of these web components, firing up a static HTML page will not display the web components. To be able to see them, you need to use a server like [web dev server](https://modern-web.dev/docs/dev-server/overview/), which I personally use.

Further addition: after downloading web dev server, it is very simple to use. Simply type `npm run serve` to get the server started. It will automatically open up a new window for you.

If you get any errors, let me know and we can figure out what's going on. 
As for lit, the main website is [here](https://lit.dev/) and [this](https://lit.dev/docs/) is the most updated documentation.

## Additional Notes
Feel free to mess around with the web component and change the style up.

Also, for some pages (like contact.html) that are really short, the footer awkwardly sits in the middle of the page. I'll probably talk to elijah or max about this, as I don't really know how we want to tackle this. 

The color, style and images used in the footer are to simply showcase the power of web components and is not meant to be the final style. Please give me ideas!

### CODE REVIEW

**Comment by GCUCoding** [comment]
Unfortunately, GitHub Pages is a static website hosting service so we cannot use this yet. I will leave this open, however, for when we can swap it over to a dynamic server hosting solution.

File: js/footer.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            flex-basis: 50%;
+            align-items: center;
+            flex-direction: column;
+
+            /* innovation. */
+            padding:22px 0px 22px 0px; 
+            
+       }
+       .footer-images {
+           /* Yes david, I know, I know.*/
```
Comment (elijaholmos) [inline js/footer.js:31]: rofl

File: contact.html [hunk 2]
```diff
@@ -53,5 +53,7 @@ <h1>Scan the code or hit the button</h1>
     >
 
     <script src="js/mobile-toggle.js" async defer></script>
+    <footer-ss></footer-ss>
```
Comment (elijaholmos) [inline contact.html:56]: @typhlosion64 would appreciate your assistance looking into the styling of the footer on this page

File: js/footer.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+		return html`<footer>
+			<div id="footer-div">
+				<p>© Grand Canyon University</p>
+				<p>
+					<a href="${imgs[0].href}" target="_blank">
+                        <img src="${imgs[0].src}" class="footer-images"/>
+                    </a>
+					<a href="${imgs[1].href}" target="_blank">
+                        <img src="${imgs[1].src}" class="footer-images"/>
+                    </a>
```
Comment (elijaholmos) [inline js/footer.js:59]: I eventually want to refactor this to use `imgs.reduce()` into a single HTML string, so we can add/remove multiple images conveniently, but I couldn't get it to work with my lack of knowledge about the Lit framework. perhaps you can poke around during a later club session

**Review by elijaholmos — APPROVED** [review]
10/10 comments. great work overall, thanks for finding this.

**Pull request merged**